keyboards/helix/rev2/keymaps/led_test/keymap.c

```diff
@@ -513,6 +513,12 @@ void music_scale_user(void)
 #ifdef SSD1306OLED
 
 void matrix_scan_user(void) {
+    static int scan_count = 0;
```

Comment:
I'm hoping to have a fix for this.  

Namely a "keyboard_init_" function that is called late enough that it doesn't get overwritten, but for now, this should be perfect.